### PR TITLE
Update to the API docs.

### PR DESCRIPTION
**Pull request by JovanVeljanoski**
Fixes some errors in the API docs.

- [x] Fix the docstring for `dataframe.fillna`
- [x] Fix the docstring  for `dataframe.first`

### CODE REVIEW

**Comment by JovanVeljanoski** [comment]
@maartenbreddels If you can, can you please fill the missing info for the `first` method quickly please.

File: packages/vaex-core/vaex/dataframe.py [hunk 1]
```diff
@@ -3875,23 +3878,34 @@ def sort(self, by, ascending=True, kind='quicksort'):
 
     @docsubst
     def fillna(self, value, fill_nan=True, fill_masked=True, column_names=None, prefix='__original_', inplace=False):
-        '''Return a DataFrame, where missing values/NaN are filled with 'value'
+        '''Return a DataFrame, where missing values/NaN are filled with 'value'.
+
+        The original columns will be renamed, and by defaul they will be hidden columns. No data is lost.
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/dataframe.py]: defaul

File: packages/vaex-core/vaex/dataframe.py [hunk 2]
```diff
@@ -522,26 +522,29 @@ def finish(counts):
...
-        :param expression: Expression or column for which to count non-missing values, or None or '*' for counting the rows
+        >>> import vaex
+        >>> df = vaex.example()
+        >>> df.first(df.x, df.y, shape=8)
+        >>> df.first(df.x, df.y, shape=8, binby=[df.y])
+        >>> df.first(df.x, df.y, shape=8, binby=[df.y])
+        array([-4.81883764, 11.65378   ,  9.70084476, -7.3025589 ,  4.84954977,
+                8.47446537, -5.73602629, 10.18783   ])
+
+        :param expression:
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/dataframe.py]: "The value to be placed in the bin."

File: packages/vaex-core/vaex/dataframe.py [hunk 3]
```diff
@@ -522,26 +522,29 @@ def finish(counts):
...
+        :param order_expression: order the values in the bins by this expression.
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/dataframe.py]: o->O (start with capital)

File: packages/vaex-core/vaex/dataframe.py [hunk 4]
```diff
@@ -522,26 +522,29 @@ def finish(counts):
...
         :param binby: {binby}
         :param limits: {limits}
         :param shape: {shape}
         :param selection: {selection}
         :param delay: {delay}
         :param progress: {progress}
         :param edges: {edges}
-        :return: {return_stat_scalar}
+        :return: The first element of a binned expression ordered by an additional expression.
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/dataframe.py]: Ndarray containing the first elements.

**Pull request merged**